README-FIPS.md

```diff
@@ -167,6 +167,18 @@ manual page.
 
  [fips_module(7)]: https://www.openssl.org/docs/manmaster/man7/fips_module.html
 
+Entropy Source
+==============
+
+The FIPS provider typically relies on an external entropy source,
+specified during OpenSSL build configuration (default: `os`).  However, by
+enabling the `enable-fips-jitter` option during configuration, an internal
+jitter entropy source will be used instead.  Note that this will cause
+the FIPS provider to operate in a non-compliant mode unless an entropy
+assessment and validation through the [CMVP] are additionally conducted.
+
```

Comment:
```suggestion
Note that that "enable-fips-jitter" option is only available for OpenSSL 3.5 or greater.
```

Reply:
I'm not sure that this is required, this README will only be distributed in 3.5 and higher...

I guess it doesn't hurt.

Reply:
I fixed the wording too :grin: 

> Note that the `enable-fips-jitter` option is only available in OpenSSL
versions 3.5 and later.